features/no_property_address.feature

```diff
@@ -0,0 +1,14 @@
+@us85a @add_property_address
```

Comment:
As the user story number tag is assigned to each scenario it probably doesn't need to be at the top of the feature file aswell.